automation/DeployToHPVS.bashlib

```diff
@@ -767,13 +767,13 @@ Passphrase: ${USERINPUT}
     #--------------------------------------------------------------------------#
     # Config variable: resource_group                                          #
     #--------------------------------------------------------------------------#
-    prompt_user "IBM Cloud Resource Group" true "default"
+    prompt_user "ID of IBM Cloud Resource Group to use for your deployment" true "default"
     change_config_var ${1} "resource_group" "${USERINPUT}"
     
     #--------------------------------------------------------------------------#
     # Config variable: resource_plan_id                                        #
     #--------------------------------------------------------------------------#
-    prompt_user "IBM Cloud Resource Plan ID" true "default"
```

Comment:
There a command somewhere that will generate all of these ids.  we should mention it here.  Something like For more information on the appropriate id use this command `ibmcloud catalog service hpvs` or stick to the default if you are unsure